tests/testutil/riofile.go

```diff
@@ -37,6 +37,24 @@ func (trf *TestRiofile) Up(t *testing.T, filename string) {
 	}
 }
 
+// Bring up a riofile by fixture file
+func (trf *TestRiofile) UpNonUnique(t *testing.T, filename, stackName string) {
+	trf.T = t
+	trf.Name = fmt.Sprintf("stack/%s/%s", testingNamespace, stackName)
```

Comment:
Should put the rest of method below this into a common method that both of them call. Or just add `stackName` as var and if empty string we do the random generation.

Reply:
Changed to perform this behavior